### PR TITLE
📜 metadata: final checks on docs and wizard

### PR DESCRIPTION
**Pull request by lucasrodes**
After latest changes to our metadata, we need to review that both `wizard` and our live documentation work and render properly.

### Changes
- `apps/wizard`: make last changes.
- `schemas`: re-order some fields so the order makes sense when in `wizard`.
- `docs/`: improve the docs, its organisation, etc.
-  `etl/docs.py`: add support to render FAQs in the future.
- other files might have been affected bc I've changes some instances of "Our World In Data" → "Our World in Data" (e.g. `libs/`)

### Test it

Basically try wizard:

```
etl-wizard
```

and check the docs

```
mkdocs serve
```

### Prior work:
- https://github.com/owid/etl/pull/1599
- https://github.com/owid/etl/issues/1584

### CODE REVIEW

File: docs/architecture/workflow/snapshot.md [hunk 1]
```diff
@@ -32,75 +32,45 @@ Note that we need a DVC file per upstream data file; hence, in some instances, i
 ## Metadata
 !!! warning "This is still being written."
```
Comment (pabloarosado) [inline docs/architecture/workflow/snapshot.md:33]: I suppose you can remove this warning.

File: docs/getting-started/index.md [hunk 2]
```diff
@@ -1,6 +1 @@
-Thanks for your interest in our data management documentation. We hope that it is helpful for you and your use case.
-
-!!! warning "Please [report any issue](https://github.com/owid/etl/issues/new?assignees=&labels=&template=docs-issue---.md) that you may find so we can keep improving our docs."
-
-
-In this section we provide the necessary material to [set up your local working environment](working-environment.md), which is a requirement to correclty build and consume datasets in our catalog. To this end, we also review how you can run some simple tests to ensure that you can actually [build datasets](building-datasets.md) and [consume them](consuming-datasets.md).
\ No newline at end of file
+In this section we provide the necessary material to [set up your local working environment](working-environment.md), which is a requirement to correclty build and consume datasets in our catalog. To this end, we also review how you can run some simple tests to ensure that you can actually [build datasets](building-datasets.md) and [consume them](consuming-datasets.md).
```
Comment (pabloarosado) [inline docs/getting-started/index.md:1]: typo correclty

File: docs/architecture/metadata/workflow.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ Metadata is added to ETL in the form of YAML files. We usually only add metadata
 ## Metadata in Snapshot
 The entrypoint to ETL is the Snapshot step. This is where we define metadata attributes for the data product of the origin. This is fundamental to ensure that the data is properly documented and that the metadata is propagated to the rest of the system.
 
-The metadata in Snapshot consists mainly of one object: `meta.origin`. To learn more about it, please refer to [the reference](../reference-origin).
+The metadata in Snapshot consists mainly of one object: `meta.origin`. To learn more about it, please refer to [the reference](../reference/origin).
```
Comment (pabloarosado) [inline docs/architecture/metadata/workflow.md:11]: This link (and I think similar ones) lead me to `http://127.0.0.1:8000/architecture/workflow/snapshot/` (which yields a 404 error), but they should lead to `http://127.0.0.1:8000/architecture/metadata/reference-origin/`.

**Review by pabloarosado — APPROVED** [review]
Looks great! There's just a minor issue. Apart from that, feel free to merge, and we can iterate on it in future PRs.
